### ISSUE TITLE
Constant CPU usage even while firefox is "idle"

### ISSUE THREAD

**Issue by xapaho** [marcelduran/yslow#59]
Constant CPU usage even while firefox is "idle"

Looking at `top` while firefox is running, but not being used :
- If YSlow! is installed and enabled, usage is constantly around 4 to 5% minimum, typically 4.3%.
- If YSlow! is not installed, or installed but disabled,  typical cpu usage is 0.3% -- as expected

YSlow 3.1.4 
firefox-16.0.2-1.fc16.x86_64

**Comment by xapaho** [comment]
NB. reproduced with a fresh profile and any system plugin disabled just in case, "of course" :s

**Comment by machixian** [comment]
Having the same isssue with Firefox 21.0 and YSlow 3.1.4.

I tried open only one empty tab in Firefox and with only one extension YSlow, top shows Firefox is using about 10% of cpu, after disable YSlow and cpu usage of Firefox is dropped blow 3%.

**Comment by marcelduran** [comment]
Does the same happen with another Firebug extension rather than YSlow?

@marcelduran

**Comment by machixian** [comment]
@marcelduran No, I have Firebug 1.11.3 installed, idle Firefox cpu usage is below 3%.

BTW: I'm using Mac OS X 10.7.5

**Comment by marcelduran** [comment]
Could you please try uninstalling YSlow and installing any other Firebug
extension just to make sure who (YSlow or Firebug extensions in general) is
causing this increase.

@marcelduran

**Comment by machixian** [comment]
Just install FireFinder, FireQuery and CSS Usage and cpu usage no noticeable change after restart Firefox. 

And I enabled YSlow once again, and cpu usage is back to 10% again after restart Firefox.

**Comment by marcelduran** [comment]
Does this 10% stick forever or only during browser/firebug startup?

On Thursday, May 16, 2013, machixian wrote:

> Just install FireFinder, FireQuery and CSS Usage and cpu usage no
> noticeable change after restart Firefox.
> 
> And I enabled YSlow once again, and cpu usage is back to 10% again after
> restart Firefox.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/marcelduran/yslow/issues/59#issuecomment-18013104
> .

## 

@marcelduran

**Comment by machixian** [comment]
The CPU usage I'm talking about is stablized CPU usage, not the CPU usage right after Firefox restarted or Firebug enabled or new page opened.

**Comment by blueyed** [comment]
Same for me, using latest YSlow, Firebug and Firefox 25 on Ubuntu (13.10).
This happens even if YSlow is the only enabled extension (Firebug disabled)!
